<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BuzzWindow</class>
 <widget class="QMainWindow" name="BuzzWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>512</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>512</width>
    <height>512</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>512</width>
    <height>512</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Meat4Eat</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/application.qrc">
    <normaloff>:/icon.ico</normaloff>:/icon.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">#centralWidget {
 border-image: url(:/buzz.png);
  background: transparent;
}
QWidget {
  color: white;
}
QLineEdit {
  background-color: rgb(101,117, 137);  
}
QPushButton {
  background-color: rgb(111,127, 147);
}
QPushButton:hover {
  background-color: rgb(151,167, 187);
}
QPushButton[enabled=&quot;false&quot;] {
  background-color: rgb(31,47,67);
}
</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>320</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="topMargin">
         <number>20</number>
        </property>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="labelTitle">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;@TITLE@&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelText">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>120</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>150</height>
             </size>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;@TEXT@&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelSource">
            <property name="text">
             <string>@SOURCE@</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>75</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="layoutFooter">
      <item>
       <spacer name="spacerLeft">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonDiscard">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>25</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Discard</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="spacerRight">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../resources/application.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButtonDiscard</sender>
   <signal>clicked()</signal>
   <receiver>BuzzWindow</receiver>
   <slot>onBtnDiscardClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>380</x>
     <y>493</y>
    </hint>
    <hint type="destinationlabel">
     <x>435</x>
     <y>511</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onBtnDiscardClicked()</slot>
  <slot>onBtnDisplayEventClicked()</slot>
 </slots>
</ui>
